### DISCUSSION TITLE
change aio to reverse proxy

### DISCUSSION THREAD

**Discussion by naftali100** [nextcloud/all-in-one#891] [Questions] [answered]
i first setup aio using the command in the readme
now i want to change it so it will run behind proxy
do i need to reinstall everything? or just delete the aio container and start it using command in reverse-proxy.md? or there is a way to change it in configs?

**Comment by szaimen** [comment, marked as answer]
> or just delete the aio container and start it using command in reverse-proxy.md?

Yes, this should be sufficient. However since there is no domain validation in your case, you need to make sure yourself that you configure the reverse proxy correctly. (Make sure to stop the containers from the aio interface before removing and recreating the mastercontainer)

**Reply by naftali100** [reply]
Will I need to restore the settings from backup after re-creating? or it's just clean replace?

**Reply by szaimen** [reply]
Which settings do you mean? It should keep all settings upon recreation except the on that you changed...
